packages/progress-bar/src/ProgressBar/ProgressBar.spec.tsx

```diff
@@ -97,6 +101,183 @@ describe('packages/progress-bar', () => {
     });
   });
 
+  describe('screen reader behavior', () => {
+    const TEST_VALUE_UNDER_50 = 43;
+    const TEST_VALUE_OVER_50 = 57;
+    const TEST_MAX_VALUE = 100;
+
+    test('does not have a live region for meter types', () => {
+      render(
+        <ProgressBar
+          type={Type.Meter}
+          value={TEST_VALUE_OVER_50}
+          maxValue={TEST_MAX_VALUE}
+        />,
+      );
+      expect(screen.queryByRole('status')).toBeNull();
+    });
+
+    test('announces initial progress, but not any further changes if next threshold is not met', () => {
+      const { rerender } = render(
+        <ProgressBar
+          type={Type.Loader}
+          isIndeterminate={false}
+          value={TEST_VALUE_UNDER_50}
+          maxValue={TEST_MAX_VALUE}
+        />,
+      );
+      expect(screen.queryByRole('status')).toHaveTextContent(
+        getAnnouncementMessage(TEST_VALUE_UNDER_50, TEST_MAX_VALUE),
+      );
+
+      rerender(
+        <ProgressBar
+          type={Type.Loader}
+          isIndeterminate={false}
+          value={TEST_VALUE_UNDER_50 + 1}
+          maxValue={TEST_MAX_VALUE}
+        />,
+      );
+      expect(screen.queryByRole('status')).toHaveTextContent(
+        getAnnouncementMessage(TEST_VALUE_UNDER_50, TEST_MAX_VALUE),
+      );
+    });
+
+    test('announces if 50% threshold passed', () => {
```

Comment:
we're technically not testing "announcement" here, just that the text of the `status` changes

Reply:
pretty sure we can't accurately test VO in jsdom